basics/data-import/upsert.md

```diff
@@ -123,6 +123,85 @@ By default, Pinot uses the value in the time column (`timeColumn` in tableConfig
 
 For partial upsert table, the out-of-order events won't be consumed and indexed. For example, for two records with the same primary key, if the record with the smaller value of the comparison column came later than the other record, it will be skipped.
 
+#### Multiple Comparison columns
+
+In some cases, especially where partial upsert might be employed, there may be multiple producers of data each writing to a mutually exclusive set of columns, sharing only the primary key. In such a case, it may be helpful to use one comparison column per producer group so that each group can manage its own specific versioning semantics without the need to coordinate versioning across other producer groups.
+
+```json
+{
+  "upsertConfig": {
+    "mode": "FULL",
```

Comment:
Should we update the example with PARTIAL upsert? Since that is the most common use case

Reply:
great call, updated 👍